packages/prisma-client/prisma/schema.prisma

```diff
@@ -68,7 +68,8 @@ model Build {
   isDev  Boolean // exctly one is true per project
   isProd Boolean // at most one is true per project (none if not published)
 
-  breakpoints Breakpoints?
+  breakpoints  Breakpoints?
+  designTokens DesignTokens?
```

Comment:
Need to make sure `designTokens` are copied to build when we publish. The code is here: https://github.com/webstudio-is/webstudio-designer/blob/fb946ab2966dfeb3ee84c2fcbd52e43f5dbfaa9b/packages/project/src/db/build.ts#L194-L266

Reply:
yes, though as long as we don't go to production with this, we should be fine, right?

Reply:
ah, yes true, I misunderstood